### PR TITLE
repo: pretend root commit exists in the root operation

### PR DESCRIPTION
**Pull request by yuja**
# Checklist

If applicable:
- [ ] I have updated `CHANGELOG.md`
- [ ] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (cli/src/config-schema.json)
- [x] I have added tests to cover my changes

### CODE REVIEW

File: lib/tests/test_commit_concurrent.rs [hunk 1]
```diff
@@ -69,9 +69,9 @@ fn test_commit_parallel(backend: TestRepoBackend) {
     // the root commit
     assert_eq!(repo.view().heads().len(), num_threads + 1);
 
-    // One additional operation for the root commit, one for initializing the repo,
-    // one for checking out the initial commit.
-    assert_eq!(count_non_merge_operations(&repo), num_threads + 3);
+    // One additional operation for the root commit, one for checking out the
+    // initial commit.
```
Comment (martinvonz) [inline lib/tests/test_commit_concurrent.rs:73]: Isn't it the operation "for the the root commit" that's no longer there? I.e., I would think this should look like this (but reflowed):

> // One additional operation for initializing the repo,
> // one for checking out the initial commit.

(Same below)

**Review by martinvonz — APPROVED** [review]
Thanks!

File: lib/tests/test_commit_concurrent.rs (hunk 1 above)
Comment (yuja) [reply lib/tests/test_commit_concurrent.rs:73]: My understanding is that this patch removes the "initialize repo" operation, so the count is decremented accordingly. It counts the number of ancestors, not the number of diffs between these.

Comment (martinvonz) [reply lib/tests/test_commit_concurrent.rs:73]: Oh, I think what confused me is the existing error that "One additional operation for the root commit" should say "... for the root *operation*", right?

Comment (yuja) [reply lib/tests/test_commit_concurrent.rs]: Right, fixed it.

**Pull request merged**